great-idea/css/index.css

```diff
@@ -63,4 +63,286 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body{ 
+	margin: 10px;
+	padding: 10px
+}
+
+header{
+	display: flex;
+}
+
+nav{
+	display: flex;
+	justify-content: space-around;
+	width: 100%;
+	margin: 10px 0;
+	padding: 20px 0;
+}
+
+a{
+	text-decoration: none;
+	color: gray;
+}
+header div{
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: 0 10px;
+	padding: 0 10px;
+}
+
+#top{
+	display: flex;
+	justify-content: space-evenly;
+	flex-wrap: wrap;
+	margin: 20px;
+	padding: 10px
+}
+
+#innovation {
+	display: flex;
+	flex-direction: column;
+	font-size: 70px;
+	justify-content: center
+}
+
+#innovation h1{
+	text-align: center;
+}
+
+button{
+	width: 150px;
+	height: 40px;
+	text-align: center;
+	margin: 0 60px
+}
+
+section, .sec-section,.section-3{
+	display: flex;
+	margin: 20px;
+	padding: 10px;
+}
+
+.section-2{
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+.middle-img{
+	margin: 20px;
+	padding: 10px;
+	width:100%;
+	display: flex;
+	justify-content: center
+}
+.middle-im{
+	width: 100%
+}
+
+.section-3 div{
+	margin: 0 20px
+}
+
+footer {
+	display: flex;
+	flex-direction: column;
+	margin: 20px;
+	padding: 20px;
+}
+
+footer h3,p{
+	font-family: sans-serif;
+}
+
+footer p{
+	text-align: center
+}
+
+hr{
+	font-weight: bolder;
+	margin: 0 30px;
+	padding: 0 20px
+}
+
+.service {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	margin: 0 30px;
+	padding: 0 0 0 40px;
+}
+
+.services{
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	margin: 30px 0;
+}
+
+.sec-service{
+	display: flex;
+	justify-content: space-around;
+	flex-wrap: wrap;
+	width: 100%;
+}
+.sec-service div{
+	flex-basis:45%;
+	display: flex;
+	flex-direction: column;
+	border: 1px solid black;
+	background-color: whitesmoke;
+	margin: 10px;
+	justify-content: space-between;
+	padding: 25px
+}
+
+button{
+	margin: 0;
+	border-radius: 5px
+}
+.card{
+	flex-direction: row;
+}
+
+.service-card {
+	display: flex;
+	width:100%;
+	justify-content: space-between;
+	height: 100%;
+	/* flex-wrap: wrap; */
+}
+
+.service-right{
+	flex-basis: 45%;
+	margin: 0 40px;
+	max-height: 100%
+}
+
+li {
+	margin: 10px;
+	list-style: square;
+}
+
+.service-left{
+	flex-basis: 47%;
+	margin: 0 20px 0 0;
+	height: 200px;
+	justify-content: space-between
+}
+
+.services-info-img{
+	width: 100%;
+	height: 300px;
+	margin: 0 50px 0 0;
+	padding: 0 20px
+}
+.p{
+	line-height: 40px
+}
+
```

Comment:
There is a better way for targeting all paragraphs in your service page then just attaching a new class to every one and each `<p>` there. Try to figure out and let me know if you find it. 
Hint: remember the `<div class="container">` I showed you yesterday. This could be added and refactored a little bit to accomplish this task!